test/helpers/ts-test.ts

```diff
@@ -17,9 +17,8 @@ const TS_MODULES_DEFAULT: TsModuleKind[] = ["4.0", "4.5", "4.8", "5.0", "5.1"];
 function getTsModuleNameWithKind(kind: TsModuleKind | undefined): string {
 	// Return the corresponding ts module
 	switch (kind) {
```

Comment:
Complete nit, feel free to ignore.
Can this switch be made to fail to build if it's not exhaustively handled?